scipy/integrate/_bvp.py

```diff
@@ -1078,6 +1087,9 @@ def solve_bvp(fun, bc, x, y, p=None, S=None, fun_jac=None, bc_jac=None,
 
         col_res, y_middle, f, f_middle = collocation_fun(fun_wrapped, y,
                                                          p, x, h)
+        bc_res = bc_wrapped(y[:, 0], y[:, -1], p)
+        max_bc_res = np.max(abs(bc_res))
```

Comment:
As `bc_res` is not necessary by itself, I think it's slightly better to eliminate one line `max_bc_res = np.max(np.abs(bc_wrapped(y[:, 0], y[:, -1], p)))` (`np.abs` just my preference). 

Another option is to use `np.linalg.norm(..., ord=np.inf)`.

But it's fine as it is.